airflow/executors/dask_executor.py

```diff
@@ -55,23 +58,28 @@ def start(self):
         else:
             security = None
 
-        self.client = distributed.Client(self.cluster_address, security=security)
+        self.client = Client(self.cluster_address, security=security)
         self.futures = {}
 
-    def execute_async(self, key, command, queue=None, executor_config=None):
-        if queue is not None:
-            warnings.warn(
-                'DaskExecutor does not support queues. '
-                'All tasks will be run in the same cluster'
-            )
+    def execute_async(self,
+                      key: TaskInstanceKeyType,
+                      command: CommandType,
+                      queue: Optional[str] = None,
+                      executor_config: Optional[Any] = None) -> None:
+        if not self.futures:
+            raise AirflowException("Executor should be started first.")
 
         def airflow_run():
             return subprocess.check_call(command, close_fds=True)
 
+        if not self.client:
+            raise AirflowException("The Dask executor has not been started yet!")
         future = self.client.submit(airflow_run, pure=False)
         self.futures[future] = key
 
-    def _process_future(self, future):
+    def _process_future(self, future: Future) -> None:
+        if not self.futures:
```

Comment:
There's mixed styles in this file -- some us `if not self.x` some do `assert self.x` -- we should be consistent

Reply:
Ah yeah. Good catch. I replaced it in in all cases where I used the NOT_STARTED message but I missed a few places where I left the original message.